### PR TITLE
FSYjbsw9: Update the code to work with the new passport-verify

### PR DESCRIPTION
**Pull request by jakubmiarka**
- renamed the response body types to reflect the matching ones
- bumped a couple of dependencies (including typescript and node)
- importing passport-verify release candidate (supporting non-matching)

### CODE REVIEW

**Comment by jakubmiarka** [comment]
A new PR will follow after successful testing to move from RC version of passport-verify and also updates to the views.

File: integration-deployment.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```
Comment (chrisholmes) [inline integration-deployment.sh]: This file shouldn't needed as we'll use concourse CF support to deploy

File: src/index.ts [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
-const serviceEntityId = process.env.ENTITY_ID || null
+const serviceEntityId = process.env.ENTITY_ID
+const matching = process.env.MATCHING || false
 
-const server = createApp(verifyServiceProviderHost, DatabaseWrapper.getDatabaseWrapper(databaseConnectionString), serviceEntityId).listen(process.env.PORT || 3200, function () {
-  console.log(`Entity Id set to ${serviceEntityId !== null ? serviceEntityId : 'null (will use verify service provider default)'}`)
-  console.log(`Stub RP app listening on port ${server.address().port}!`)
-})
+if (matching) {
+  console.log(`Starting the app in a matching mode`)
```
Comment (chrisholmes) [inline src/index.ts]: we probably should use 'debug' for this logging

File: src/views/view-identity-attributes.njk [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>First Name</td>
+            <td><code>{{identity.attributes.firstName.value}}</code></td>
+        </tr>
+        {% for attribute in identity.attributes.middlenames %}
+          <tr>
+              <td><code>Middle Name</td>
+              <td><code>{{attribute.value}}</code></td>
```
Comment (chrisholmes) [inline src/views/view-identity-attributes.njk]: Could we add whether the attribute is `verified` here?

File: integration-manifest.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+memory: 256M
```
Comment (chrisholmes) [inline integration-manifest.yml]: This should be covered the CI pipeline.

File: integration-deployment.sh (hunk 1 above)
Comment (jakubmiarka) [reply integration-deployment.sh]: yep, will now remove.. we were testing before concourse was there

File: src/views/view-identity-attributes.njk (hunk 3 above)
Comment (jakubmiarka) [reply src/views/view-identity-attributes.njk]: done

File: integration-manifest.yml (hunk 4 above)
Comment (jakubmiarka) [reply integration-manifest.yml]: yep, will now remove.. we were testing before concourse was there

File: src/index.ts (hunk 2 above)
Comment (jakubmiarka) [reply src/index.ts]: Then it's not going to show in the console. I think this is useful to see when starting the app (using `./startup` for instance)

**Pull request merged**